fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file "" # Path to the json secret file - Follow https://github.com/fastlane/supply#setup to get one
```

Comment:
Is this needed? As supply `uploads app metadata, screenshots and binaries to Google Play` we won't use it no?

Reply:
You are right, I'll remove it